common/src/models/error.rs

```diff
@@ -19,13 +18,11 @@ impl ApiError {
 }
 
 impl Display for ApiError {
+    // TODO: styling was removed, create pretty print method in cargo-shuttle?
```

Comment:
Todo

Reply:
The reason I removed the styling is we need to convert to `ApiError` to check the status code in `gateway` `impl IntoResponse for Error`, and I was then recording the `ApiError` in the error event. However, we can clone the gateway error kind, create the `ApiError` from that, and still emit the original error in the error event. So I restored the styling on `ApiError`.